hack/run-functests.sh

```diff
@@ -25,9 +27,20 @@ echo "Focusing on $FOCUS"
 
 export SUITES_PATH=cnf-tests/bin
 
-if ! cnf-tests/test-run.sh -ginkgo.focus=$FOCUS -junit /tmp/artifacts/ -report /tmp/artifacts/; then
-  failed=true
-  failures+=( "Tier 2 tests for $FEATURES" )
+if [ $CNF_TESTS_IMAGE != "" ]; then
+  cp -f $KUBECONFIG _cache/kubeconfig
+  echo "Running dockerized version via $CNF_TESTS_IMAGE"
+
+  if ! $IMAGE_RUN_CMD run -v $(pwd)/_cache/:/kubeconfig -v /tmp/artifacts/:/reports -e KUBECONFIG=/kubeconfig/kubeconfig $CNF_TESTS_IMAGE /usr/bin/test-run.sh \
+       -ginkgo.focus $FOCUS -junit /reports/ -report /reports/; then
+    failed=true
+    failures+=( "Tier 2 tests for $FEATURES" )
+  fi
+else
+  if ! cnf-tests/test-run.sh -ginkgo.focus=$FOCUS -junit /tmp/artifacts/ -report /tmp/artifacts/; then
+    failed=true
+    failures+=( "Tier 2 tests for $FEATURES" )
+  fi
```

Comment:
maybe better unify these so:
if [ $CNF_TESTS_IMAGE ]; then
    EXEC_TESTS=$IMAGE_RUN_CM run -v ....
else
    EXEC_TESTS=cnf-tests/test-run.sh ...
fi

if ! $EXEC_TESTS; then
    failed=true
    failures+=( "Tier 2 tests for $FEATURES" )
fi

which are the same in both cases..

Reply:
Will do.